### PR TITLE
IN16B QENS: relax too strict error

### PR DESCRIPTION
**Pull request by gvardany**
**Description of work.**

This PR relaxes an error to warning in IN16B QENS reduction, as requested.

**Report to:** M. Gonzalez. 

**To test:**

This should now only log warnings instead of raising an error and stopping.
```
ws = IndirectILLReductionQENS(Run='233277.nxs', BackgroundRun='233002.nxs')
```

Files are below, crop `.zip`.

[233002.nxs.zip](https://github.com/mantidproject/mantid/files/2179524/233002.nxs.zip)
[233277.nxs.zip](https://github.com/mantidproject/mantid/files/2179525/233277.nxs.zip)

No issue, no notes.

Does this update require release notes?
- [ ] Yes
- [x] No

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionQENS.py [hunk 1]
```diff
@@ -271,7 +271,7 @@ def _warn_negative_integral(self, ws, message):
         for item in mtd[tmp_int]:
             for index in range(item.getNumberHistograms()):
                 if item.readY(index)[0] <= 0:
-                    raise RuntimeError('Negative or 0 integral in spectrum #{0} {1}'.format(index,message))
+                    self.log().warning('Negative or 0 integral in spectrum #{0} {1}'.format(index,message))
```
Comment (KarlPalmen) [inline Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionQENS.py]: This warning could be output many times.

Comment (gvardany) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionQENS.py]: That's expected. It is output for every spectra that gets negative.

**Comment by martyngigg** [comment]
@KarlPalmen Are you reviewing this?

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionQENS.py (hunk 1 above)
Comment (KarlPalmen) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionQENS.py]: OK

**Comment by KarlPalmen** [comment]
I'll go ahead with this.

**Comment by KarlPalmen** [comment]
The attached zip files are not recognised as valid by my computer (Windows 7).

**Comment by gvardany** [comment]
There should be .nxs files, did you cut the .zip from the names?

**Comment by KarlPalmen** [comment]
I've just spotted the instruction to rename without `.zip` and have followed it.

**Review by KarlPalmen — APPROVED** [review]

**Pull request merged**